akka-bench-jmh/src/main/scala/akka/actor/ActorCreationWithReflectionCacheBenchmark.scala

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (C) 2014-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.actor
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+/*
+without caching:
+[info] ActorCreationWithReflectionCacheBenchmark.synchronousStarting    ss    3  243.775 ± 330.837  us/op
+
+with caching:
+[info] ActorCreationWithReflectionCacheBenchmark.synchronousStarting    ss    3  482.001 ± 1494.089  us/op
+
```

Comment:
us/op with caching is larger and bigger variance... are you sure this is improving things?

Reply:
Sorry I was doing other things when I am running the bench. I read through the doc of sbt-jmh and re-run it.